### PR TITLE
feat: release apisix docker image by tags

### PR DESCRIPTION
**Pull request by oldthreefeng**
Usege

```
git tag apisix-2.7 && git push origin --tags apisix-2.7
```

build logs: 

https://github.com/oldthreefeng/apisix-docker/actions/runs/970587183

if apisix-dashboard use this action. 

i can also make the changes like. ( on the way)

```
git tag dashboard-2.7 && git push origin --tags dashboard-2.7
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,3 +81,10 @@ Tips: If there is a port conflict, please modify the host port through `docker r
 ```shell
 $ docker run -v `pwd`/all-in-one/apisix/config.yaml:/usr/local/apisix/conf/config.yaml -v `pwd`/all-in-one/apisix-dashboard/conf.yaml:/usr/local/apisix-dashboard/conf/conf.yaml -p 19080:9080 -p 12379:2379 -p 19000:9000 -d apache/apisix-dashboard:whole
 ```
+
+## release apisix docker image
+
+
```
Comment (tokers) [inline README.md:87]: Only one empty line is enough.

Comment (oldthreefeng) [reply README.md]: done

**Review by Yiyiyimu — CHANGES_REQUESTED** [review]
Hi @oldthreefeng Actually I don't think there is an advantage to replace `branch` with `tag` for triggering the workflow. The reason I chose `branch` but not `tag` before is that, maintainer would create a [PR](https://github.com/apache/apisix-docker/pull/188) (so with a branch) for the new release. So there would not be another step to push to docker hub.

This time I forgot to inform the maintainer of the correct step for the new release. Would try to remember that next time 😿

**Comment by oldthreefeng** [comment]
ok

**Pull request closed without merging**